migrations/curate-qdm-1.1.0-qdm-1.2.0.toml

```diff
@@ -0,0 +1,72 @@
+new_repo = "../ocsf/qdm"
```

Comment:
Can we keep them in separate folders like these three files can be moved to qdm-1.1.0-qdm-1.2.0